docs/source/getting_started/02_configure_pkdk.md

```diff
@@ -75,11 +75,12 @@ By default, `model.yolo` detects humans. We can change its behavior either by 1)
                                  'output.media_writer': {'output_dir': '<directory to save results>'}}"
  ```
```

Comment:
Possible to remove the backslashes? Users might think they are needed.

Reply:
Think its okay. Made the necessary changes.

Sphinx and Github adds a scroll bar:
![image](https://user-images.githubusercontent.com/35787241/129585892-f387cc8b-590b-4b2c-a1ec-62a118541d7a.png)
![image](https://user-images.githubusercontent.com/35787241/129586148-c57dadcf-c20f-405f-9b2e-a9a0ef11ef74.png)